modules/reindex/src/test/java/org/opensearch/index/reindex/ReindexRestClientSslTests.java

```diff
@@ -90,7 +90,7 @@
 
     @BeforeClass
     public static void setupHttpServer() throws Exception {
-        InetSocketAddress address = new InetSocketAddress(InetAddress.getLoopbackAddress().getHostAddress(), 0);
+        InetSocketAddress address = new InetSocketAddress("localhost", 0);
```

Comment:
The solution is pretty simple and has no consequences: just use `localhost` as bind and connect address, otherwise the certificate validation is going to fail in any case.